llvm/docs/AMDGPUUsage.rst

```diff
@@ -520,6 +520,106 @@ Every processor supports every OS ABI (see :ref:`amdgpu-os`) with the following
 
      =========== =============== ============ ===== ================= =============== =============== ======================
 
+Generic processors also exist. They group multiple processors into one,
+allowing to build code once and run it on multiple targets at the cost
+of less features being available.
+
+Generic processors are only available on Code Object V6 and up.
+
```

Comment:
Document that generic processes have a version (see :ref:`amdgpu-elf-header-e_flags-table-v6-onwards`) and explain how it is used by the runtime.